containers/kubernetes/how-to/revoke-kubeconfig.mdx

```diff
@@ -0,0 +1,72 @@
+---
+meta:
+  title: How to revoke a kubeconfig of a Kapsule cluster
+  description: This page explains how to revoke a kubeconfig of a Kapsule cluster
+content:
+  h1: How to revoke a kubeconfig of a Kapsule cluster
+  paragraph: This page explains how to revoke a kubeconfig of a Kapsule cluster
+tags: kubernetes kubernetes-kapsule kapsule cluster kubeconfig revoke iam
+dates:
+  validation: 2023-11-28
+  posted: 2023-11-28
+categories:
+  - kubernetes
+---
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
+</Message>
+
+By the end of 2023, a new Authentication (Auth) system, based on IAM tokens, is introduced. **This results in the coexistence of two authentication mechanisms.**
+
+Previously, users were limited to static admin tokens provisioned by Kapsule.
+
+In the new system, when a user loses access rights (e.g., leaves the organization), the Kubernetes administrator can simply modify the IAM group, edit its policy, or remove the user to revoke cluster access.
+This approach is a derivation from the previous method, where the owner had to reset the admin token, impacting all other users and requiring actions through the Console, CLI, or API methods.
+Now, users have the alternative option of using a key provisioned by IAM for authentication.
+
+## How to revoke a kubeconfig file with IAM
+
+To revoke cluster access, you can have to ensure that the API key associated with it is no longer granted permission.
```

Comment:
can have -> have ?
is no longer granted permission -> no longer has permission ?